lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -541,7 +541,7 @@ export class UserAgentApplication {
             }
 
             // On Fulfillment
-            const responseType: string = isLoginCall ? ResponseTypes.id_token : this.getTokenType(account, request.scopes, false);
+            const responseType: string = this.getTokenType(account, request.scopes);
```

Comment:
Why did we get rid of the `isLoginCall` logic here?

Reply:
The idea was to make the entire decision based on the scopes and not which API was called. We couldn't remove the account factor from the decision because we need to support the specific use case of the account passed in being different from the one in the cache. We were able to remove isLoginCall by having the API validate and manage the scopes set before this point is reached, which shifts the responsibility from the `isLoginCall` boolean to the state of the scopes.

Reply:
I re-added the ternary check for `isLoginCall`. There's no way to tell if it's a login call based on scopes only, seems like this was a moot effort, but I still think the new version of this code is an improvement. Thanks!